<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventana_nuevareceta</class>
 <widget class="QDialog" name="ventana_nuevareceta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nueva receta</string>
  </property>
  <widget class="QPushButton" name="btn_cancelar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>440</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="direccion_imagen">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>390</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="descripcion">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>311</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Imagen:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ingredientes">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Descripción: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingredientes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>País</string>
   </property>
  </widget>
  <widget class="QToolButton" name="carga_imagen">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>390</y>
     <width>23</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pais">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>311</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_aceptar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombre">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>311</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="preparacion">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Preparación: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
